dash_docs/tutorial/performance.py

```diff
@@ -178,6 +178,28 @@ def large_params_function(largeValue1, largeValue2):
 )
     '''),
 
+    reusable_components.Markdown('''
+#### Inline Clientside Callbacks
+
+You can also write the JavaScript code directly in your Dash app by
+supplying a value to the "source" argument. Here's an example using
+the same function as above:
+    '''),
+
+    Syntax('''
+from dash.dependencies import Input, Output
+
+app.clientside_callback(
+    Output('out-component', 'value'),
+    [Input('in-component1', 'value'), Input('in-component2', 'value')],
+    source="""
+    function(largeValue1, largeValue2) {
+        return someTransform(largeValue1, largeValue2);
+    }
+    """
+)
```

Comment:
Doesn't the function need to be first? Per the docstring:

https://github.com/plotly/dash/blob/38994ebca24775d6b588051bef6a54699c1eb1ff/dash/dash.py#L1249-L1261

```suggestion
app.clientside_callback(
    """
    function(largeValue1, largeValue2) {
        return someTransform(largeValue1, largeValue2);
    }
    """,
    Output('out-component', 'value'),
    [Input('in-component1', 'value'), Input('in-component2', 'value')]
)
```